AdyenActions/UI/View Controllers/DelegatedAuthentication/DelegatedAuthenticationErrorView.swift

```diff
@@ -52,6 +53,50 @@ internal final class DelegatedAuthenticationErrorView: UIView {
         scopeInstance: self
     )
     
+    internal lazy var troubleshootingTitle: UILabel = .init(
+        style: style.troubleshootingTitleStyle,
+        accessibilityPostfix: "troubleshootingTitle",
+        multiline: false,
+        textAlignment: .center,
+        scopeInstance: self
+    )
+    
+    internal lazy var troubleshootingDescription: UILabel = .init(
+        style: style.troubleshootingDescriptionStyle,
+        accessibilityPostfix: "troubleshootingDescription",
+        multiline: true,
+        textAlignment: .center,
+        scopeInstance: self
+    )
+    
+    internal lazy var troubleShootingButton: SubmitButton = {
+        let button = SubmitButton(style: self.style.troubleshootingButtonStyle)
+        button.addTarget(self, action: #selector(troubleShootingButtonTapped), for: .touchUpInside)
+        button.accessibilityIdentifier = ViewIdentifierBuilder.build(scopeInstance: self, postfix: "TroubleshootingButton")
```

Comment:
Let's keep suffixes consistent, it's lowercase in the line 66 and it's uppercase here.

Reply:
Thanks, typo. Trouble Shoot 😆 Corrected.